config/dev.webpack.config.js

```diff
@@ -1,11 +1,14 @@
-/* global require, module, __dirname */
+/* global */
 const { resolve } = require('path');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 const config = require('@redhat-cloud-services/frontend-components-config');
 const { config: webpackConfig, plugins } = config({
   rootFolder: resolve(__dirname, '../'),
   debug: true,
 });
+const federationPlugin = require('./federation-plugin');
+
+plugins.push(federationPlugin);
```

Comment:
I'd merge this into one line
```suggestion
plugins.push(require('./federation-plugin'));
```